### PR TITLE
Delay relay browser startup until SVN Update finished

### PR DESCRIPTION
**Pull request by jaadams5**
I thought about it.  Changed the logic so it didn't need two wait loops and lowered the wait to 30 seconds.

### CODE REVIEW

File: src/net/sourceforge/kolmafia/webui/RelayLoader.java [hunk 1]
```diff
@@ -36,20 +39,32 @@ protected RelayLoader(final String location, final boolean isRelayLocation) {
     }
   }
 
+  private void waitForSVNUpdateToFinish() {
+    PauseObject pauser = new PauseObject();
+    int triesLeft = 6;
+    while ((triesLeft > 0) && LoginManager.isSvnLoginUpdateUnfinished()) {
```
Comment (gausie) [inline src/net/sourceforge/kolmafia/webui/RelayLoader.java:45]: Why not make this a for loop like the loop below?

Comment (jaadams5) [reply src/net/sourceforge/kolmafia/webui/RelayLoader.java:45]: The "loop below" was existing code that I didn't think about changing.  I tend to use a for when the number of times I do something is important and a while when there is some condition that I expect to change.  I can certainly make a decision and be consistent.  Stand by.

File: src/data/defaults.txt [hunk 2]
```diff
@@ -120,6 +120,7 @@ global	relayAddsUseLinks	true
 global	relayAddsWikiLinks	false
 global	relayAllowRemoteAccess	false
 global	relayBrowserOnly	false
+global  relayDelayForSVN    5
```
Comment (midgleyc) [inline src/data/defaults.txt:123]: you've used spaces instead of tabs here.

**Comment by jaadams5** [comment]
I reworked the pause loops and added a preference.  5 seconds is a little too short on my system so it might as well be a preference and not hardwired.

**Review by midgleyc — CHANGES_REQUESTED** [review]

File: src/net/sourceforge/kolmafia/webui/RelayLoader.java [hunk 3]
```diff
@@ -36,22 +37,40 @@ protected RelayLoader(final String location, final boolean isRelayLocation) {
     }
   }
 
+  private void pauseAndWaitForRelayAction(int timeInMilliseconds, boolean waitForStart) {
+    // Pause for input value of seconds, waking up to check relay server
```
Comment (jaadams5) [inline src/net/sourceforge/kolmafia/webui/RelayLoader.java:41]: waitForStart is always true in this code but I have an uncommitted test that, if I keep, used the same routine to  wait for shutdown.  Not sure it's a good test but the parameter here defers the discussion.

File: src/data/defaults.txt (hunk 2 above)
Comment (jaadams5) [reply src/data/defaults.txt:123]: will fix.  how did you find that so fast

Comment (midgleyc) [reply src/data/defaults.txt]: if you view it on GitHub it looks obviously misaligned, and then I checked by highlighting the characters.

**Review by midgleyc — APPROVED** [review]

**Pull request merged**